src/main/java/javax/validation/spi/ConfigurationState.java

```diff
@@ -81,6 +82,31 @@
 	Set<InputStream> getMappingStreams();
 
 	/**
+	 * Returns a set of value extractors.
+	 * <p>
+	 * The extractors are retrieved from the following sources in decreasing
+	 * order:
+	 * <ul>
+	 *     <li>extractors passed programmatically to {@link Configuration}</li>
+	 *     <li>extractors defined in {@code META-INF/validation.xml} provided
+	 *     that {@code ignoredXmlConfiguration} is {@code false}.</li>
```

Comment:
This is a VERY minor thing, but IMO either all list items should end with a period or none of them.